storage/volume/types/volume.go

```diff
@@ -83,10 +83,12 @@ type VolumeConfig struct {
 
 // VolumeSpec represents volume spec.
 type VolumeSpec struct {
+	ClusterAPI    string   `json:"clusterapi"`
```

Comment:
Quite confused on the naming of this field. `ClusterAPI` is some kind of network address?
If that, how about adding `ClusterAPIAddress` ?